client/src/main.jsx

```diff
@@ -0,0 +1,19 @@
+import { createRoot } from 'react-dom/client'
+import App from './App.jsx'
+import './index.css'
+import './i18.js'
+import { Auth0Provider } from '@auth0/auth0-react';
+
+const root = createRoot(document.getElementById('root'));
+
+root.render(
+<Auth0Provider
+    domain="dev-2mlojserqc0ytr8b.us.auth0.com"
+    clientId="NyjTsxGX6tAxOxKL6JkjzR6hY7MKhHn4"
```

Comment:
how are `domain` and `clientId` values chosen?

Reply:
They are not being choosen, they are being generated from my `auth0 dashboard` for `auth` purposes.

Reply:
Please add a reference to documentation how `Auth0` key generation works